jumpscale/sals/zos/__init__.py

```diff
@@ -1,4 +1,34 @@
-def export_module_as():
+__all__ = ["get", "clear_cache"]
+
+INSTANCES = {}  # Dict[identityName, Zosv2Instance]
+
+
+def get(identity_name=None):
+    """Gets a Zosv2 instance for identity `identity_name`
+
+    Args:
+        identity_name ([str], optional): [identity name]. Defaults to None, if None then it returns a Zosv2 instance relative to the me identity
+
+    Returns:
+        [type]: [description]
+    """
+    global INSTANCES, ME_INSTANCE
+    from jumpscale.loader import j
     from .zos import Zosv2
 
-    return Zosv2()
+    if not identity_name:
+        identity_name = j.core.identity.me.instance_name
```

Comment:
I think If I have no default identity this one gonna crash ?

Reply:
Addressed, also handled the case of using identity_name that doesn't exist